test_files/declare_class_overloads/externs.js

```diff
@@ -130,9 +130,9 @@ OverloadValueOf.prototype.valueOf = function() {};
 function Merged() {}
 
 /**
- * @param {(string|number|!Array<!OverloadBigMix>)} a_or_c_or_e_or_f
+ * @param {(string|number|!Array<!OverloadBigMix>|!Array<!OverloadBigMix>)} a_or_c_or_e_or_f
```

Comment:
I recall at some point in some code we grab the set of union elements and drop duplicates.  I guess it's a different code path than here...